components/tyk-enterprise-portal/values.yaml

```diff
@@ -90,6 +90,14 @@ resources: {}
 securityContext:
   runAsUser: 1000
   fsGroup: 2000
+containerSecurityContext:
```

Comment:
Can you add a comment above field to explain what it does.(Even though name is self explanatory!)